pt-BR/mail/chrome/global-platform/win/nsWindowsHooks.properties

```diff
@@ -1,6 +1,6 @@
-yesButtonLabel=Yes
-noButtonLabel=No
-cancelButtonLabel=Cancel
-checkBoxLabel=Check at startup next time, too.
-promptText=%S is not currently set as your default browser. Would you like to make it your default browser?
-prefsLabel=Pr&eferences
+yesButtonLabel=Sim
+noButtonLabel=Não
+cancelButtonLabel=Cancelar
+checkBoxLabel=Verificar ao iniciar na próxima vez.
```

Comment:
CORRECTIONS NEEDED it should be +checkBoxLabel=Também verificar ao iniciar na próxima vez.